umm-lib/src/cmr/umm/related_url_helper.clj

```diff
@@ -30,7 +32,9 @@
 (defn documentation-url?
   "Returns true if the related-url is documentation url"
   [related-url]
-  (some #{(:mime-type related-url)} DOCUMENTATION_MIME_TYPES))
+  (let [mime-type (:mime-type related-url)]
+    (when (seq mime-type)
```

Comment:
I can't think of another place where we use `seq` on a string. I think it would work, but we should make this `(when mime-type` or really as suggested above `(when-let [mime-type (:mime-type related-url)]`